pkg/minikube/cluster/ip.go

```diff
@@ -47,11 +47,7 @@ func HostIP(host *host.Host, clusterName string) (net.IP, error) {
 		}
 		return net.ParseIP(ip), nil
 	case driver.KVM2:
-		ip, err := host.Driver.GetIP()
-		if err != nil {
-			return []byte{}, errors.Wrap(err, "Error getting VM/Host IP address")
-		}
-		return net.ParseIP(ip), nil
+		return net.ParseIP("192.168.39.1"), nil
```

Comment:
are u sure this is doing the right thing ? what if the KVM user has multiple minikube profiles, will they have same IP ?

Reply:
It was hardcoded before, but we missed to add the .1 at the end when making it more generic: d48b0a58e

Reply:
Fixed this. I wrote as the same as the vmware case.